src/postamble.js

```diff
@@ -316,6 +316,10 @@ Module['exit'] = Module.exit = exit;
 var abortDecorators = [];
 
 function abort(what) {
+  if (Module['onRuntimeInitializeFailed']) {
```

Comment:
if we want this callback to be just for runtime initialization failures, then we should erase `Module['onRuntimeInitializeFailed']` after initialization succeeds (so aborts later on don't cause it to be called).

alternatively, if you want a general callback when the problem aborts for any reason, including after initialization, then we should rename this. (or, perhaps we should just recommend people set Module.abort to this callback? right now we'd override that, but we could change that)